lib/scryptenc/scryptenc.c

```diff
@@ -135,6 +135,11 @@ pickparams(size_t maxmem, double maxmemfrac, double maxtime,
 		maxN = opslimit / (*r * 4);
 		for (*logN = 1; *logN < 63; *logN += 1) {
 			checkN = (uint64_t)(1) << *logN;
+
+			/*
+			 * We compare against maxN / 2 because logN is one
```

Comment:
In retrospect this seems obvious, but I think the comment wouldn't hurt because I'm bound to forget about it and then get puzzled again in a year or two.